utils/src/math.rs

```diff
@@ -34,13 +34,17 @@ mod tests {
             (3, 2),
             (9, 4),
             (15, 4),
+            (16, 4),
+            (17, 5),
             (15430, 14),
             (usize::MAX, 64),
         ];
         for (d, expected_res) in tests.iter() {
             let res = ceil_log2(*d);
-            println!("ceil(log2({d})) = {res}, expected = {expected_res}");
-            assert!(res == *expected_res)
+            assert_eq!(
```

Comment:
didn't know we had our own `ceil_log2` function lol, I've been using `i.log2()` all along.